src/styles/blocks/card.scss

```diff
@@ -0,0 +1,94 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
```

Comment:
Thats correct - you need to set this to make sure all works correctly
Still, a better idea is to reset `box-sizing` for all elements unless you have a reason not to do it (having in mind you set same border-box value for several elements)

https://css-tricks.com/box-sizing/#aa-good-better-and-probably-best-box-sizing-reset-methods